src/CalcViewModel/ApplicationViewModel.cpp

```diff
@@ -204,3 +207,59 @@ void ApplicationViewModel::SetMenuCategories()
     // property setter logic.
     Categories = NavCategoryGroup::CreateMenuOptions();
 }
+
+void ApplicationViewModel::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e, float width, float height)
```

Comment:
I would suggest to better separate View and ViewModel:
- Don't pass `sender` or `RoutedEventArgs` (these 2 items are related to the View)
- Use a name describing what the method does and not what is the reason for the call. Something like `SwitchViewModel()` for example

Reply:
@joseartrivera, what's your opinion?

Reply:
Agreed, looks like the last commit updated the args, just needs a reasonable name. Something like EnterAlwaysOnTop().